zenodo/modules/spam/views.py

```diff
@@ -125,17 +126,34 @@ def delete(user_id):
         return render_template('zenodo_spam/delete.html', **ctx)
 
 
-def _evaluate_user_domain(email):
-    email_domain = email.rsplit('@', 1)[-1].lower()
+def normalize_emails(email):
+    email = email.lower()
+    username = email[: email.index("@")]
+    no_dots_username = username.replace(".", "")
+    domain = email[email.index("@") :]
```

Comment:
Let's normalize the domain as well l, i.e. googlemail.com should become gmail.com